lib/netsuite/records/vendor_bill_item_list.rb

```diff
@@ -0,0 +1,32 @@
+module NetSuite
+  module Records
+    class VendorBillItemList
```

Comment:
This is perfect for now, but it would be great to use the Sublist implementation to DRY some of the list code up going forward:

https://github.com/NetSweet/netsuite/blob/master/lib/netsuite/support/sublist.rb

Ex:

https://github.com/NetSweet/netsuite/blob/c29cffe0a13326e0e2283f29e516a8ca5f588b72/lib/netsuite/records/contact_access_roles_list.rb